### PR TITLE
feat: Add new output called env_vars for Redis

### PR DESCRIPTION
**Pull request by q2w**
This PR makes below changes,

1. Add new output called env_vars for Redis
2. Update metadata files using `make generate_docs`

### CODE REVIEW

File: modules/redis-cluster/metadata.yaml [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+kind: BlueprintMetadata
+metadata:
+  name: terraform-google-memorystore
+  annotations:
+    config.kubernetes.io/local-config: "true"
+spec:
+  info:
+    title: Memorystore Redis Cluster Terraform Module
+    source:
+      repo: https://github.com/q2w/terraform-google-memorystore.git
```
Comment (bharathkkb) [inline modules/redis-cluster/metadata.yaml:25]:
```suggestion
      repo: https://github.com/terraform-google-modules/terraform-google-memorystore.git
```

File: modules/memcache/metadata.yaml [hunk 2]
```diff
@@ -19,106 +19,109 @@ metadata:
...
-    description: The memorystore instance ID.
-  - name: nodes
-    description: Data about the memcache nodes
-  - name: region
-    description: The region the instance lives in.
-  roles:
-  - level: Project
+  info:
+    source:
+      repo: https://github.com/q2w/terraform-google-memorystore.git
```
Comment (bharathkkb) [inline modules/memcache/metadata.yaml:24]:
```suggestion
      repo: https://github.com/terraform-google-modules/terraform-google-memorystore.git
```

**Review by bharathkkb — APPROVED** [review]
Looks like there is a diff post apply. We may have to do skip default verify on that test https://github.com/terraform-google-modules/terraform-google-memorystore/blob/550e009156eb420a9b87b4397a5f96abfa57c245/test/integration/redis-cluster/redis_cluster_test.go#L29

File: modules/redis-cluster/metadata.yaml (hunk 1 above)
Comment (q2w) [reply modules/redis-cluster/metadata.yaml]: Done. Thanks!

**Review by bharathkkb — APPROVED** [review]

**Pull request merged**